.storybook/themes/news.scss

```diff
@@ -911,6 +912,75 @@
 				double-chain-child-item-empty: (
 					display: contents,
 				),
+				full-author-bio: (
+					display: flex,
+					flex-direction: column,
+					gap: var(--global-spacing-4),
+					margin: auto,
+					max-width: 30rem,
```

Comment:
We should not be setting a max-width on the block even if `30rem` is pretty wide

Reply:
```suggestion
```